ESP_ATMod/command.cpp

```diff
@@ -2559,6 +2638,13 @@ bool readNumber(unsigned char *inpBuf, uint16_t &offset, uint32_t &output)
 {

```

Comment:
I understand you need to read negative numbers. But the output variable should be then `int32_t`. I don't know how this change affects the current code. There may jump dozens of unsigned/signed warnings. 
I'd prefer to detect the '-' in the only place where you read negative numbers (line 761) as mandatory character.

Reply:
Resolved.